htdocs/commande/card.php

```diff
@@ -166,6 +166,8 @@
 		}
 	}
 
+	$selectedLines = "";
```

Comment:
selectedLines should be an array.
Is   selectedLines = array()    solve the trouble ?

Reply:
the main problem is the variable is not defined initialy. propose to define it as array if you prefer